### DISCUSSION TITLE
Screen Size

### DISCUSSION THREAD

**Discussion by NitrusYT1** [CosmosOS/Cosmos#1850] [Q&A]
How do i change the screen size

**Comment by ghost** [comment]
change the screen size using what graphic mode?

**Reply by partisani** [reply]
console mode lmao

**Reply by ghost** [reply]
As far as I am aware, you cannot. I think it is stuck to 80 by 25 in default console mode.
There are some custom VGA drivers out there which can switch to 90*60 for example.

**Reply by quajak** [reply]
I havnt tested it but `VGAScreen.SetTextMode` might be able to change other console modes.

**Reply by prescott2m** [reply]
> I havnt tested it but `VGAScreen.SetTextMode` might be able to change other console modes.

It Works.. But It Cant Make Up Its Mind... the string outputted is 90 chars big
![image](https://user-images.githubusercontent.com/49623720/140232047-555b7dcf-7452-4513-a4e4-71ed170e498b.png)

**Reply by quajak** [reply]
As stated in  https://github.com/CosmosOS/Cosmos/issues/1984  it looks like only half of the necessary code to support other ScreenSizes is currently implemented.
